python/lsst/obs/base/defineVisits.py

```diff
@@ -635,7 +635,7 @@ def run(
         dataIds : `Iterable` [ `dict` or `~lsst.daf.butler.DataCoordinate` ]
             Exposure-level data IDs.  These must all correspond to the same
             instrument, and are expected to be on-sky science exposures.
-        collections : Any, optional
+        collections : `Sequence` [ `str` ] or `str` or `None`
```

Comment:
Do you also need to change the typing for `_buildVisitRecords` which uses Any at the moment? And compute viit regions compute method?

Reply:
I was trying to quickly fix the mypy error I'm getting. I take your point, but I'm worried that's going to snowball. I'll give it a _quick_ try though...